mobile.html

```diff
@@ -8,9 +8,14 @@
     <title>Mobile portofolio</title>
 </head>
 <body>
+    <div class="page1">
```

Comment:
- [ ] Kindly remove the default margin given to the `body` by the browser. You can see that the `body` has `8px` in the design. You can easily fix this by overwriting it with `margin: 0` in your CSS.

Reply:
when I add any attribute value with zero got linter errors. what should I do?